docs/root/intro/version_history.rst

```diff
@@ -90,6 +90,7 @@ Version history
 * config: Fixed stat inconsistency between xDS and ADS implementation. :ref:`update_failure <config_cluster_manager_cds>`  
   stat is incremented in case of network failure and :ref:`update_rejected <config_cluster_manager_cds>` stat is incremented 
   in case of schema/validation error.
+* cli: Added support for :ref:`component log level <operations_cli>` command line option for configuring log levels of individual components.
```

Comment:
Please alpha order this (and fix others if you see them).

Reply:
Sorry. I did not know it has to be in alpha order. I was under the impression that we should just add at the end. Read the pull request template it spells that out very clearly. I have added this entry in correct order. Will push another PR to fix other's as well.

Reply:
OK sorry about that. Do you mind fixing the PR template also?

Reply:
Sorry, I meant pull request template is clear on alpha order. I missed it. Will send PR for others...